foundations/foundation-vertx/src/test/java/org/apache/servicecomb/foundation/vertx/http/TestReadStreamPart.java

```diff
@@ -186,7 +187,7 @@ public void saveToFile() throws InterruptedException, ExecutionException, IOExce
 
     part.saveToFile(file.getAbsolutePath()).get();
 
-    Assert.assertEquals(src, FileUtils.readFileToString(file));
+    Assert.assertEquals(src, FileUtils.readFileToString(file, Charset.defaultCharset()));
```

Comment:
replace all defaultCharset to UTF_8?

Reply:
done. replace all defaultCharset with UTF_8